### PR TITLE
Docs: Plugin-tools - Config and migration - chunk 7

### PR DESCRIPTION
**Pull request by josmperez**
### What is this feature?
This PR holds a review of plugins docs, focusing on structure, editorial soundness, and conformity to doc team style guidelines.

### Why do we need this feature?
This fulfills the doc improvement initiative described in the plugins doc plan.

### Who is this feature for?
Plugin developers

### CODE REVIEW

File: docusaurus/docs/advanced-configuration.md [hunk 1]
```diff
@@ -3,21 +3,17 @@ id: advanced-configuration
 title: Advanced Configuration
 ---
 
-The `.config/` directory holds the preferred configuration for the different tools used to develop, test, and build a plugin.
+The plugin `.config/` directory holds the preferred configuration for the different tools used to develop, test, and build a Grafana plugin. Although you can make changes, we recommend against doing so. Instead, follow the guidance in this topic to customize your tooling configs.
```
Comment (leventebalogh) [inline docusaurus/docs/advanced-configuration.md:6]: Are we sure we need the word "plugin" here before ".config/"? (I think the .config directory doesn't closely relate to the plugin "itself", it is more like generated build configuration, so the plugins can be packaged by the plugin devs with no effort.)

File: docusaurus/docs/advanced-configuration.md [hunk 2]
```diff
@@ -3,21 +3,17 @@ id: advanced-configuration
...
 
 :::danger
 
-To make future updates easier do **not** edit files in the `.config` directory. Instead, follow the directions in this article to customise the tooling configurations.
+Customize the plugin tooling at your own risk. If you choose to extend the tooling configurations, then you may experience issues such as failure to compile or load in Grafana.
```
Comment (leventebalogh) [inline docusaurus/docs/advanced-configuration.md:10]: I would probably keep the mention of the `.config` folder in this heads-up message. The key point we would like to make is that ideally they shouldn't touch the config files under `.config/`, however they are free to customize them in a bit safer way by following our extensions below.

**Review by mckn — APPROVED** [review]
LGTM!

File: docusaurus/docs/advanced-configuration.md (hunk 1 above)
Comment (academo) [inline docusaurus/docs/advanced-configuration.md:6]: I understand the change of voice but IMO this is not warning enough that they must not edit the `.config` folder. The original language clearly stayed "do not edit". the "at your own risk" sounds more like a legal term and us devs usually work better with explicit text.

File: docusaurus/docs/advanced-configuration.md (hunk 2 above)
Comment (academo) [inline docusaurus/docs/advanced-configuration.md:10]: I see you swapped the next sentence to here. These are warning about two different hings.

The first warning was about editing the files inside `.config` and the second warning is about extending the configuration. In general terms we don't want devs editing anything inside the `.config` folder and extending the configuration at their own risk.

File: docusaurus/docs/migrating-from-toolkit.mdx [hunk 3]
```diff
@@ -3,26 +3,24 @@ id: migrating-from-toolkit
...
-- **Better documentation:** The documentation for @grafana/create-plugin is more comprehensive and easier to discover than @grafana/toolkit.
-
-:::warning
-Before running the following command we strongly suggest backing up the code. Ideally, store the plugin code in a Git repository and run the command on a clean branch to easily identify changes made by the migration command.
-:::
+- **More flexible:** With `create-plugin`, you have more control over your plugins and its dependencies. The additional control translates into greater ease of customizing the tooling to fit your specific needs.
+- **Faster development:** With its out-of-the-box development environment, `create-plugin` can significantly reduce development time compared to using `grafana-toolkit`.
+- **Improved testing capabilities:** Easily test plugins with `create-plugin` GitHub workflows that automate unit and e2e test runs whenever you push changes to GitHub. 
+- **Better documentation:** The documentation for `create-plugin` is more comprehensive and easier to discover than `grafana-toolkit`.
```
Comment (academo) [inline docusaurus/docs/migrating-from-toolkit.mdx:17]: Could you please add a new point here staying that the old tooling `grafana-toolkit` is deprecated and we won't offer support any longer?

**Comment by sympatheticmoose** [comment]
Taken a look, agree with comments from other reviewers above and added one additional suggestion 👍

File: docusaurus/docs/advanced-configuration.md (hunk 1 above)
Comment (josmperez) [reply docusaurus/docs/advanced-configuration.md:6]: Fixed.

File: docusaurus/docs/advanced-configuration.md (hunk 2 above)
Comment (josmperez) [reply docusaurus/docs/advanced-configuration.md:10]: Fixed.

File: docusaurus/docs/migrating-from-toolkit.mdx (hunk 3 above)
Comment (josmperez) [reply docusaurus/docs/migrating-from-toolkit.mdx:17]: Fixed.

File: docusaurus/docs/advanced-configuration.md (hunk 1 above)
Comment (josmperez) [reply docusaurus/docs/advanced-configuration.md:6]: Fixed.

File: docusaurus/docs/advanced-configuration.md (hunk 2 above)
Comment (josmperez) [reply docusaurus/docs/advanced-configuration.md:10]: Fixed.

**Review by leventebalogh — APPROVED** [review]
LGTM

**Review by sympatheticmoose — APPROVED** [review]

**Review by academo — APPROVED** [review]

**Pull request merged**